tracsubtickets/web_ui.py

```diff
@@ -98,10 +100,8 @@ def _append_parent_links(self, req, data, ids):
     def prepare_ticket(self, req, ticket, fields, actions):
         pass
 
-    def get_children(self, parent_id, db=None):
+    def get_children(self, parent_id, db):
         children = {}
-        if not db:
-            db = self.env.get_db_cnx()
         cursor = db.cursor()
         cursor.execute("SELECT parent, child FROM subtickets WHERE parent=%s",
                        (parent_id, ))
```

Comment:
The following would conform to the Trac coding style:
```
for child, in self.env.db_query("""
        SELECT child FROM subtickets WHERE parent=%s
        """, (parent_id, )):
    children[child] = None
```

I won't note all the places a change like this could be made, but there are many.